assembl/graphql/idea.py

```diff
@@ -298,6 +298,13 @@ class Meta:
     posts = SQLAlchemyConnectionField(
         'assembl.graphql.post.PostConnection',  # use dotted name to avoid circular import  # noqa: E501
         random=graphene.Boolean())
+    thematic = graphene.Field(lambda: Thematic)
+
+    def resolve_thematic(self, args, context, info):
```

Comment:
isn't it more `resolve_parent_idea` or `resolve_parent_thematic` ?

Reply:
I'm ok with simply `resolve_thematic` here. A question is associated to one thematic, implicitly a parent.

Reply:
@amensouissi Please add a Python test for this.

Reply:
oh ok, my bad, it is the question schema here (I relied on `idea.py` and assumed it was a idea/thematic)